calDAV Binding not working under 2.0.0

Under an earlier version of openhab 2.0.0 it has worked. But now it does not.
I would like to fix it but don't know how. The methods exists, but OSGI says it they does not exist. 
Why?
```
19:16:45.126 [ERROR] [org.openhab.binding.caldav-command  ] - [org.openhab.binding.caldav_command(42)] unbind method [unsetCalDavLoader] not found; Component wi
ll fail
19:16:45.127 [ERROR] [org.openhab.binding.caldav-command  ] - [org.openhab.binding.caldav_command(42)] unbind method [unsetCalDavLoader] not found
19:16:45.131 [ERROR] [org.openhab.binding.caldav-command  ] - [org.openhab.binding.caldav_command(42)] bind method [addBindingProvider] not found
19:16:45.132 [ERROR] [org.openhab.binding.caldav-command  ] - [org.openhab.binding.caldav_command(42)] unbind method [removeBindingProvider] not found
19:16:45.133 [ERROR] [org.openhab.binding.caldav-command  ] - [org.openhab.binding.caldav_command(42)] unbind method [unsetCalDavLoader] not found
19:16:45.151 [ERROR] [org.openhab.binding.caldav-command  ] - [org.openhab.binding.caldav_command(42)] bind method [addBindingProvider] not found
19:16:45.152 [ERROR] [org.openhab.binding.caldav-command  ] - [org.openhab.binding.caldav_command(42)] unbind method [removeBindingProvider] not found
19:16:45.152 [ERROR] [org.openhab.binding.caldav-command  ] - [org.openhab.binding.caldav_command(42)] unbind method [unsetCalDavLoader] not found
19:16:45.154 [ERROR] [org.openhab.binding.caldav-personal ] - [org.openhab.binding.caldav_personal(44)] bind method [addBindingProvider] not found; Component wi
ll fail
19:16:45.157 [ERROR] [org.openhab.binding.caldav-personal ] - [org.openhab.binding.caldav_personal(44)] bind method [addBindingProvider] not found
19:16:45.164 [ERROR] [org.openhab.binding.caldav-personal ] - [org.openhab.binding.caldav_personal(44)] unbind method [unsetCalDavLoader] not found; Component w
ill fail
19:16:45.165 [ERROR] [org.openhab.binding.caldav-personal ] - [org.openhab.binding.caldav_personal(44)] unbind method [unsetCalDavLoader] not found
19:16:45.167 [ERROR] [org.openhab.binding.caldav-personal ] - [org.openhab.binding.caldav_personal(44)] unbind method [removeBindingProvider] not found; Compone
nt will fail
19:16:45.168 [ERROR] [org.openhab.binding.caldav-personal ] - [org.openhab.binding.caldav_personal(44)] unbind method [removeBindingProvider] not found
19:16:45.169 [ERROR] [org.openhab.binding.caldav-personal ] - [org.openhab.binding.caldav_personal(44)] bind method [addBindingProvider] not found
19:16:45.170 [ERROR] [org.openhab.binding.caldav-personal ] - [org.openhab.binding.caldav_personal(44)] unbind method [unsetCalDavLoader] not found
19:16:45.170 [ERROR] [org.openhab.binding.caldav-personal ] - [org.openhab.binding.caldav_personal(44)] unbind method [removeBindingProvider] not found
```

>  I do not see any of the log messages

This is not completely right, I am (as expected) still seeing:
```
22:11:52.785 [ERROR] [org.openhab.binding.caldav-command  ] - [org.openhab.binding.caldav_command(285)] unbind method [unsetCalDavLoader] not found; Component will fail
22:11:52.786 [ERROR] [org.openhab.binding.caldav-command  ] - [org.openhab.binding.caldav_command(285)] unbind method [unsetCalDavLoader] not found
```

--- a/bundles/binding/org.openhab.binding.caldav-command/src/main/java/org/openhab/binding/caldav_command/internal/CalDavBinding.java
+++ b/bundles/binding/org.openhab.binding.caldav-command/src/main/java/org/openhab/binding/caldav_command/internal/CalDavBinding.java
@@ -95,7 +95,7 @@ public class CalDavBinding extends AbstractBinding<CalDavBindingProvider>impleme
         this.calDavLoader.addListener(this);
     }
 
-    public void unsetCalDavLoader() {
+    public void unsetCalDavLoader(CalDavLoader calDavLoader) {
         this.calDavLoader.removeListener(this);
         this.calDavLoader = null;
     }

--- a/bundles/binding/org.openhab.binding.caldav-personal/src/main/java/org/openhab/binding/caldav_personal/internal/CalDavBinding.java
+++ b/bundles/binding/org.openhab.binding.caldav-personal/src/main/java/org/openhab/binding/caldav_personal/internal/CalDavBinding.java
@@ -64,7 +64,7 @@ public class CalDavBinding extends AbstractBinding<CalDavBindingProvider>impleme
         this.calDavLoader.addListener(this);
     }
 
-    public void unsetCalDavLoader() {
+    public void unsetCalDavLoader(CalDavLoader calDavLoader) {
         this.calDavLoader.removeListener(this);
         this.calDavLoader = null;
     }

--- a/bundles/io/org.openhab.io.caldav/src/main/java/org/openhab/io/caldav/internal/CalDavLoaderImpl.java
+++ b/bundles/io/org.openhab.io.caldav/src/main/java/org/openhab/io/caldav/internal/CalDavLoaderImpl.java
@@ -117,9 +117,11 @@ public class CalDavLoaderImpl extends AbstractActiveService implements ManagedSe
     }
 
     private void removeAllJobs() throws SchedulerException {
-        scheduler.deleteJobs(new ArrayList<JobKey>(scheduler.getJobKeys(jobGroupEquals(JOB_NAME_EVENT_RELOADER))));
-        scheduler.deleteJobs(new ArrayList<JobKey>(scheduler.getJobKeys(jobGroupEquals(JOB_NAME_EVENT_START))));
-        scheduler.deleteJobs(new ArrayList<JobKey>(scheduler.getJobKeys(jobGroupEquals(JOB_NAME_EVENT_END))));
+        if(scheduler!=null) {
+            scheduler.deleteJobs(new ArrayList<JobKey>(scheduler.getJobKeys(jobGroupEquals(JOB_NAME_EVENT_RELOADER))));
+            scheduler.deleteJobs(new ArrayList<JobKey>(scheduler.getJobKeys(jobGroupEquals(JOB_NAME_EVENT_START))));
+            scheduler.deleteJobs(new ArrayList<JobKey>(scheduler.getJobKeys(jobGroupEquals(JOB_NAME_EVENT_END))));
+        }
     }
 
     @Override

